pkg/controllers/dynakube/activegate/internal/proxy/reconciler.go

```diff
@@ -56,18 +57,18 @@ func (r *Reconciler) generateForDynakube(ctx context.Context, dynakube *dynatrac
 		return errors.WithStack(err)
 	}
 
-	coreLabels := kubeobjects.NewCoreLabels(dynakube.Name, kubeobjects.ActiveGateComponentLabel)
-	secret, err := kubeobjects.CreateSecret(r.scheme, r.dynakube,
-		kubeobjects.NewSecretNameModifier(capability.BuildProxySecretName(dynakube.Name)),
-		kubeobjects.NewSecretNamespaceModifier(r.dynakube.Namespace),
-		kubeobjects.NewSecretLabelsModifier(coreLabels.BuildMatchLabels()),
-		kubeobjects.NewSecretTypeModifier(corev1.SecretTypeOpaque),
-		kubeobjects.NewSecretDataModifier(data))
+	coreLabels := labels.NewCoreLabels(dynakube.Name, labels.ActiveGateComponentLabel)
+	secret, err := k8sobjectsecret.CreateSecret(r.scheme, r.dynakube,
```

Comment:
```suggestion
	secret, err := k8sobjectsecret.Create(r.scheme, r.dynakube,
```

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/b58b302251d3905a66492b1b0993d5ee3d7f2669